crates/libs/registry/src/value_iterator.rs

```diff
@@ -79,7 +80,8 @@ impl<'a> Iterator for ValueIterator<'a> {
                             Value::String(String::new())
                         } else {
                             let value = unsafe {
-                                std::slice::from_raw_parts(
+                                // TODO: Vec<u8> does not guarantee alignment for u16
```

Comment:
Let's create an issue to avoid forgetting about this TODO.

Reply:
#3048

Reply:
Thanks!